lmdeploy/messages.py

```diff
@@ -277,6 +277,8 @@ class PytorchEngineConfig:
             If unspecified, will use the default version.
         quant_policy (int): default to 0. When k/v is quantized into 4 or 8
             bit, set it to 4 or 8, respectively
+        distributed_executor_backend (str): backend of distributed backend,
+            options: ['mp', 'ray']
```

Comment:
what's `mp` short for?

Reply:
multiprocessing.
https://github.com/vllm-project/vllm/blob/44c33f01f309099066d358b1f51f0d0a703b46ca/vllm/config.py#L1404